test/e2e/e2e_suite_test.go

```diff
@@ -72,10 +73,12 @@ func TestMain(m *testing.M) {
 				return ctx, err
 			}
 
-			// Load docker image into kind
-			log.Println("Loading docker image into kind cluster...")
-			if err := kindCluster.LoadImage(ctx, dockerImage); err != nil {
-				log.Printf("Failed to load image into kind: %s", err)
+			// LoadImage does not work when using podman, so LoadImageArchive is used instead
+			// refer to https://github.com/kubernetes-sigs/kind/issues/2038
```

Comment:
Pls move this comment into `LoadContainerImageToKindCluster` (right following the` if containerTool == "podman"`)